modules/dbnd-airflow/setup.py

```diff
@@ -13,15 +13,15 @@
 
 requirements_for_airflow = [
     "WTForms<2.3.0",  # fixing ImportError: cannot import name HTMLString at 2.3.0
-    "Werkzeug<1.0.0",
+    "Werkzeug<1.0.0,>=0.15.0",
     "cattrs==1.0.0",  # airflow requires ~0.9 but it's py2 incompatible (bug)
     "psycopg2>=2.7.4,<2.8",
 ]
 
 setuptools.setup(
     name="dbnd-airflow",
     package_dir={"": "src"},
-    install_requires=["dbnd==" + version],
+    install_requires=["dbnd==" + version, "packaging"],
```

Comment:
I'll approve this for the current PR (because it really is a dependency), and I'll add a task to vendorize this package (we only use single function from this package in versioned_dagbag.py)

Reply:
ok